docs/policy/dynamic-pools.md

```diff
@@ -0,0 +1,122 @@
+   - The number of CPUs allocated in each dynamic-pool is the sum of the requests of the containers in the dynamic pool and the CPUs allocated based on the CPU utilization of the workload.
+8. When a new container is created on a Kubernetes node, the policy first decides the type of the dynamic-pool that will run the container. The decision is based on the annotation of the pod, or the namespace if annotations are not given.
+
+## Deployment
+
+### Install cri-resmgr
+
+Deploy cri-resmgr on each node as you would for any other policy. See [installation](https://intel.github.io/cri-resource-manager/stable/docs/installation.html) for more details.
+
+## Configuration
+
+The dynamic-pools policy is configured using the yaml-based configuration system of CRI-RM. See [setup and usage](https://intel.github.io/cri-resource-manager/stable/docs/setup.html#setting-up-cri-resource-manager) for more details on managing the configuration.
+
+### Parameters
+
+Dynamic-pools policy parameters:
+
+* `PinCPU` controls pinning a container to CPUs of its dynamic-pool. The default is  `true`: the container cannot use other CPUs.
+* `PinMemory` controls pinning a container to the memories that are closest to the CPUs of its dynamic-pool. The default is `true`: allow using memory only from the closest NUMA nodes. Warning: this may cause kernel to kill workloads due to out-of-memory error when closest NUMA nodes do not have enough memory. In this situation consider switching this option `false`.
+* `ReservedPoolNamespaces` is a list of namespaces (wildcards allowed) that are assigned to the special reserved dynamic-pool, that is, will run on reserved CPUs. This always includes the `kube-system` namespace.
+* `DynamicPoolTypes` is a list of dynamic-pool type definitions. Each type can be configured with the following parameters:
+  - `Name` of the dynamic-pool type. This is used in pod annotations to assign containers to dynamic-pool of this type.
+  - `Namespaces` is a list of namespaces (wildcards allowed) whose pods should be assigned to this dynamic-pool type, unless overridden by pod annotations.
+  - `CpuClass` specifies the name of the CPU class according to which CPUs of dynamic-pools are configured.
+  - `AllocatorPriority` (0: High, 1: Normal, 2: Low, 3: None). CPU allocator parameter, used when creating new or resizing existing dynamic-pools.
+
+Related configuration parameters:
+
+* `policy.ReservedResources.CPU` specifies the (number of) CPUs in the special `reserved` dynamic-pool. By default all containers in the `kube-system` namespace are assigned to the reserved dynamic-pool.
+* `policy.AvailableResources.CPU` specifies the CPUs that can be used by the policy, including `policy.ReservedResources.CPU`.
+* `cpu.classes` defines CPU classes and their parameters (such as `minFreq`, `maxFreq`, `uncoreMinFreq` and `uncoreMaxFreq`).
+
+### Example
+
+```yaml
+apiVersion: v1
```

Comment:
The last change request from my side before approval: would you change this yaml, too, to the format that can be directly copy-pasted to a configuration file, and that it will then just work with:
```
cri-resmgr -force-config copypasted.cfg
```

Reply:
> The last change request from my side before approval: would you change this yaml, too, to the format that can be directly copy-pasted to a configuration file, and that it will then just work with:
> 
> ```
> cri-resmgr -force-config copypasted.cfg
> ```

Hi, @askervin , I have modified this yaml to be able to work through `cri-resmgr -force-config copypasted.cfg`, can you please check again.